packages/components/addon/components/hds/button/index.js

```diff
@@ -126,25 +124,6 @@ export default class HdsButtonIndexComponent extends Component {
     }
   }
 
-  /**
```

Comment:
removed the @type argument from the API in favour of the type native attribute (this simplifies the code)